.github/workflows/doxygen.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         doxyfile-path: "./Doxyfile" # default is ./Doxyfile
         # Working directory
         working-directory: "." # default is .
+        # Dependency for creating UML class/collaboration diagrams
+        additional-packages: graphviz
```

Comment:
I don't think "additional-packages: graphviz" has any effect inside the docker container which we use now; however graphviz is already installed there :-)